engine/table/src/main/java/io/deephaven/engine/table/impl/util/MemoryTableLoggers.java

```diff
@@ -24,12 +25,22 @@
 public class MemoryTableLoggers {
```

Comment:
I think we should rename this to something else.  PerformanceMetrics, or DeephavenMetrics or something like that